src/pages/lib/auth/delete_user/q/platform/[platform].mdx

```diff
@@ -1,23 +1,99 @@
 export const meta = {
-  title: `Delete user`,
-  description: `Delete a user`
+  title: `Delete user account`,
+  description: `Learn how to delete a user account.`
 };
 
-import ios0 from '/src/fragments/lib/auth/native_common/delete_user/common.mdx';
+<InlineFilter filters={["js", "react-native"]}>
 
-<Fragments fragments={{ ios: ios0 }} />
+Empowering users to delete their account can improve trust and transparency. You can programmatically enable self-service account deletion with Amplify Auth. Users can delete their own account through your app interface by confirming intent before final removal. In this guide we will review how you can enable this for your users.
 
-import flutter1 from '/src/fragments/lib/auth/native_common/delete_user/common.mdx';
+Before you begin, you will need:
+- An Amplify project with the Auth category configured
+- The Amplify Libraries installed and configured
+- A test user to delete
 
-<Fragments fragments={{ flutter: flutter1 }} />
+## Allow my users to delete their account
+
+You can quickly set up account deletion for your users with the Amplify Libraries. Invoking the `deleteUser` API to delete a user from the Auth category will also sign out your user. 
+
+You should also be aware of which data this deletion includes and does not include. Depending on your application, data associated with your user may be stored in other resources such as Amazon DynamoDB or Amazon S3. It is recommended that you carefully evaluate this data and implement code to remove or otherwise sanitize it, if necessary, prior to giving users the ability to delete their own user record.
```

Comment:
The first sentence seems uneccessary to me. So maybe just this?

Depending on your application, data associated with your user may be stored in other resources such as Amazon DynamoDB or Amazon S3. It is recommended that you carefully evaluate this data and implement code to remove or otherwise sanitize it, if necessary, prior to giving users the ability to delete their own user record.

Reply:
also - the info here seems to be a duplicate of the callout. I think we should keep one or the other - and maybe specifically the callout.